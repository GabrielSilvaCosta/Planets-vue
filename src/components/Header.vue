<template>
    <div class="header">
        <h1>Solar System</h1>
    </div>

</template>

<script>
export default {
  name: "Header",
};
</script>

<style>
.header {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100px;
  background-color: #333;
  color: #fff;
  font-size: 24px;
  font-weight: bold;
  text-align: center;
  text-transform: uppercase;
  letter-spacing: 4px;
  margin-bottom: 20px;
}
</style>
