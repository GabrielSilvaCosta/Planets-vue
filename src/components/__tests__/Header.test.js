import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";
import Header from "../Header.vue";

describe("Header", () => {
    it("should render correctly", () => {
        const wrapper = mount(Header, {
            props: {
                title: "Teste Header",
            },
        });
        expect(wrapper.html()).toMatchSnapshot();
    });
})