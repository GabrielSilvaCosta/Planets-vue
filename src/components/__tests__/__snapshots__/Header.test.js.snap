// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > should render correctly 1`] = `
"<div class="header" title="Teste Header">
  <h1>Solar System</h1>
</div>"
`;
